navbar.ts

```diff
@@ -24,6 +24,11 @@ export const navbar: NavbarConfig = [
                 link: '/supported-devices/',
             },
             '/guide/installation/',
+            {
```

Comment:
I think it's better to move this under the Configuration section and call it "All settings" (having both Configuration and Settings is a bit confusing)

Reply:
The current docs provide more context about the settings, therefore I think its better to keep that as the main thing.